datasets/modeled/transport/dataset.json5

```diff
@@ -1,59 +1,94 @@
 {
-  dataset_id: "tempo_conus_2022",
+  dataset_id: "decarb_2023_transport",
   dataset_type: "modeled",
-  data_source: "tempo",
-  sector_description: "transportation",
-  description: "CONUS 2022 TEMPO county-level results for PEV charging.",
+  data_source: "tempo evix polaris",
+  sector_description: "electric load associated with light, medium, and heavy-duty road transportation",
+  description: "DECARB 6 results from TEMPO, EVI-X, and POLARIS (Nov 2023) with profiles using 2018 as the weather year",
   origin_creator: "Arthur Yip",
   origin_organization: "NREL",
   origin_contributors: [
     "Arthur Yip",
+    "Brennan Borlaug"
   ],
-  origin_project: "",
-  origin_date: "",
-  origin_version: "",
-  source: "",
+  origin_project: "DECARB 6",
+  origin_date: "2023-11-30",
+  origin_version: "0.1.0",
+  source: "TEMPO, EVI-X, and POLARIS simulations for DECARB 6",
```

Comment:
You could add software versioning details here if those are available. For an example: https://github.com/dsgrid/dsgrid-project-DECARB/blob/main/datasets/modeled/buildings/dataset.json5#L21

Reply:
not available